src/protobuf/bmv2/table.proto

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020 Google LLC
```

Comment:
Are we ever gonna use any of these files individually? If not, have you considered combining them into a single file?
If these files all serve a single purpose, having them in a single file with a comment at the top explaining the file's purpose may make it easier to digest for a reader.

As a general principle, we optimize for code readability over ease of writing the code.

Reply:
I already suggested this and there is a bug to do this. It'll be done in a separate CL :)